jwst/extract_1d/extract.py

```diff
@@ -2518,9 +2520,8 @@ def separate_target_and_background(self, ref):
         return (mask_target, mask_bkg)
 
 
-def run_extract1d(input_model, refname, smoothing_length, bkg_order,
-                  log_increment, subtract_background, apply_nod_offset,
-                  was_source_model=False):
+def run_extract1d(input_model, extract_ref_name, smoothing_length, bkg_order, log_increment, subtract_background,
```

Comment:
I know that when adding a new argument to a function it's obviously simplest to just append it to the end of the list, but wouldn't it make more sense (stylistically or whatever) to put extract_ref_name and apcorr_ref_name back-to-back in the list somewhere? No impact on functionality of course, just API style.